src/main/java/com/puppycrawl/tools/checkstyle/site/XdocsTemplateSink.java

```diff
@@ -62,4 +67,11 @@ public void body_() {
         flush();
         init();
     }
+
+    @Override
+    public void link(String name) {
```

Comment:
I need to mention in Javadoc that I added this method because the default `XdocSink` adds a `class="externalLink"` that I don't want.

This is a trimmed-down version of the default `Xhtml5BaseSink`
https://github.com/apache/maven-doxia/blob/24ab8ba193ec44007333775576277f9aa9ea165f/doxia-core/src/main/java/org/apache/maven/doxia/sink/impl/Xhtml5BaseSink.java#L1572-L1598